### PR TITLE
Issue 1927: Skip publish for npm and nuget

### PR DESCRIPTION
**Pull request by j4ckofalltrades**
Skip publishing non-Maven based artifacts
    
Refs strongbox/strongbox#1927

This is based off of the `issue-1927-nuget-examples` branch in #25.

### CODE REVIEW

File: .github/workflows/gh-registry.yml [hunk 1]
```diff
@@ -2,6 +2,36 @@ name: Publish strongbox-examples to the GitHub Package Registry
...
+        with:
+          dotnet-version: '3.1.x'
+          source-url: https://nuget.pkg.github.com/strongbox/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: mcs -t:library -out:HelloWorld.dll ./src/HelloWorld.cs
+        working-directory: 'hello-strongbox-nuget-mono'
+      - run: nuget pack ./Hello.Strongbox.Nuget.Mono.nuspec
+        working-directory: 'hello-strongbox-nuget-mono'
+      - run: nuget push *.nupkg -SkipDuplicate
```
Comment (j4ckofalltrades) [inline .github/workflows/gh-registry.yml:33]: From [NuGet CLI push command docs](https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-push):

> -SkipDuplicate
> 
> (5.1+) If a package and version already exists, skip it and continue with the next package in the push, if any.

File: .github/workflows/gh-registry.yml [hunk 2]
```diff
@@ -30,7 +60,13 @@ jobs:
           scope: '@strongbox'
       - run: npm install
         working-directory: 'hello-strongbox-npm'
-      - run: npm publish
+      - run: |
```
Comment (j4ckofalltrades) [inline .github/workflows/gh-registry.yml:63]: `npm info <existing_package_name>` returns a non-empty string (existing package's info).

File: .github/workflows/gh-registry.yml (hunk 1 above)
Comment (carlspring) [reply .github/workflows/gh-registry.yml:33]: Fantastic! Thanks! :)

**Comment by carlspring** [comment]
Thanks for these follow-up fixes! :)

**Pull request merged**